plugins/analogix/fu-analogix-device.c

```diff
@@ -241,7 +241,24 @@ fu_analogix_device_probe (FuDevice *device, GError **error)
 		g_prefix_error (error, "failed to find update interface: ");
 		return FALSE;
 	}
-
+        guint16 vid = fu_usb_device_get_vid (FU_USB_DEVICE (device));
+	if (vid == 0x1F29)
+	{
+		fu_device_set_vendor (device, "Analogix Semiconductor Inc.");
+	}
+	else if (vid == 0x050D)
+	{
+		fu_device_set_vendor (device, "Belkin");
+	}
+	else if (vid == 0x047D)
+	{
+		fu_device_set_vendor (device, "Kensington");
+	}
+	else if (vid == 0x0502)
+	{
+		fu_device_set_vendor (device, "Acer");
+	}
+	else{}
```

Comment:
Do you need an of these? The vendor should be bieng set automatically using the udev vendor database.

Reply:
Do you mean I don't need call fu_device_set_verdor to set vendor name? But if I didn't call this API, I found verdor is displayed as "Analogix".

Reply:
Try removing the manual:

    > plugins/analogix/fu-analogix-device.c:  fu_device_set_vendor(FU_DEVICE(self), "Analogix Semiconductor Inc.");

Reply:
I removed this line codes, but the vendor displays "Analogix".

Reply:
What does the USB descriptor say? i.e. the string pointed by iManufacturer?

Reply:
I see, let me check this, I will pull a new request, thank you!